cumulus_library_data_metrics/data_metrics/q_term_use/q_term_use.py

```diff
@@ -22,70 +23,55 @@ def add_metric_queries(self) -> None:
         self.make_table(
             src="Immunization",
             field="vaccineCode",
-            systems=[
-                # The FHIR spec also gives urn:oid:1.2.36.1.2001.1005.17 as an example,
-                # but the US Core profile drops that suggestion in favor of only CVX.
-                "http://hl7.org/fhir/sid/cvx",
-            ],
+            # The FHIR spec also gives urn:oid:1.2.36.1.2001.1005.17 as an example,
+            # but the US Core profile drops that suggestion in favor of only CVX.
+            systems=[systems.CVX],
         )
         self.make_table(
             src="Medication",
             field="code",
-            systems=[
-                # The FHIR spec gives SNOMED as an example, but the US Core profile drops that
-                # suggestion in favor of only RxNorm
-                "http://www.nlm.nih.gov/research/umls/rxnorm",
-            ],
+            # The FHIR spec gives SNOMED as an example, but the US Core profile drops that
+            # suggestion in favor of only RxNorm
+            systems=[systems.RXNORM],
         )
         self.make_table(
             src="MedicationRequest",
             field="medicationCodeableConcept",
-            systems=[
-                # The FHIR spec gives SNOMED as an example, but the US Core profile drops that
-                # suggestion in favor of only RxNorm
-                "http://www.nlm.nih.gov/research/umls/rxnorm",
-            ],
+            # The FHIR spec gives SNOMED as an example, but the US Core profile drops that
+            # suggestion in favor of only RxNorm
+            systems=[systems.RXNORM],
         )
-        self.make_table(src="Observation", field="code", systems=["http://loinc.org"])
+        self.make_table(src="Observation", field="code", systems=[systems.LOINC])
         self.make_table(
             src="Observation",
             field="valueCodeableConcept",
-            systems=[
-                # Base FHIR doesn't suggest anything specific here, but the Laboratory and
-                # Smoking Status profiles both want SNOMED.
-                "http://snomed.info/sct",
-            ],
+            # Base FHIR doesn't suggest anything specific here, but the Laboratory and
+            # Smoking Status profiles both want SNOMED.
+            systems=[systems.SNOMED],
         )
         self.make_table(
             src="Procedure",
             field="code",
             systems=[
                 # Base FHIR only gives SNOMED as an example,
                 # but the US Core v4 profile lists all these.
-                "http://loinc.org",
-                "http://snomed.info/sct",
+                systems.CPT,
+                systems.LOINC,
+                systems.SNOMED,
                 "http://www.ada.org/cdt",
-                "http://www.ama-assn.org/go/cpt",
                 "https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets",
                 "http://www.cms.gov/Medicare/Coding/ICD10",
```

Comment:
These remaining ones seem very one-off, so I didn't bother with making a big deal of em.